pynicotine/gtkgui/widgets/clipboard.py

```diff
@@ -25,12 +26,25 @@
 """ Clipboard """
 
 
+if GTK_API_VERSION >= 4:
+    _clipboard = Gdk.Display.get_default().get_clipboard()
+else:
+    _clipboard = Gtk.Clipboard.get(Gdk.SELECTION_CLIPBOARD)
```

Comment:
It might be better to free the Clipboard object assignment after each use?

Reply:
GTK keeps a reference to it too, so it doesn't matter.